mappings/net/minecraft/entity/damage/DamageEffects.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_8107 net/minecraft/entity/damage/DamageEffects
+	FIELD field_42281 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_42282 id Ljava/lang/String;
+	FIELD field_42283 sound Lnet/minecraft/class_3414;
+	METHOD <init> (Ljava/lang/String;ILjava/lang/String;Lnet/minecraft/class_3414;)V
+		ARG 3 id
+		ARG 4 sound
+	METHOD method_48786 sound ()Lnet/minecraft/class_3414;
```

Comment:
Actually, shouldn't this be `getSound`? We don't tend to go for record-style naming for getters.

Reply:
Probably. However, I'm going away for the weekend so it might take a while to get it fixed.